game/Items.gd

```diff
@@ -95,8 +96,11 @@ func generate_items(tier_level, count=DEFAULT_COUNT):
 			item_list.append(item)
 	return item_list
 
+func generate_boss_key():
+	var item = get_item_by_id(ItemList.BOSS_KEY)
+	return LootItem.new(LootType.GEAR, item)
```

Comment:
Should also handle setting the flag of _floor_key to true after generating the key

Reply:
And remove where I set it in overworld after calling combat.

Reply:
Done with new push.

Reply:
I don’t see a change

Reply:
I'm doing the work in a different location:
![image](https://user-images.githubusercontent.com/10472087/100416716-be16b280-3044-11eb-8f1a-409a5ec56c5b.png)